tutorial/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    ?= -E -W
+SPHINXOPTS    ?= -E -W -j 8
```

Comment:
This speeds up local building of tutorials, which is nice since we now have 50+ tutorials